fdbserver/DataDistribution.actor.cpp

```diff
@@ -6001,20 +6025,27 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self,
 				    .detail("PriorityTeam1Left", 0)
 				    .detail("PriorityTeam0Left", 0)
 				    .detail("PrioritySplitShard", 0)
-				    .trackLatest("MovingData");
+				    .trackLatest("MovingData"); // this track event's lifestyle is contolled by movingDataEventHolder in
+				                                // DataDistribution.actor.cpp. Please make sure the keys in both places
+				                                // are matched.
```

Comment:
This and the two below should use the tracking key, as I believe you have access to it here.